build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -849,15 +851,14 @@ CONFIG_LWIP_DHCPS_MAX_STATION_NUM=8
 CONFIG_NET_SECURITY_TLS=y
 CONFIG_TLS_WITH_HW_ACCEL=y
 CONFIG_TLS_MPI_MAX_SIZE=512
-# CONFIG_TLS_HAVE_NO_TIME_DATE is not set
 
 #
 # HW Options
 #
-# CONFIG_TLS_HW_RNG is not set
-# CONFIG_TLS_HW_DH_PARAM is not set
-# CONFIG_TLS_HW_ECDH_PARAM is not set
-# CONFIG_TLS_HW_ECDSA_VERIFICATION is not set
+CONFIG_TLS_HW_RNG=y
+CONFIG_TLS_HW_DH_PARAM=y
+CONFIG_TLS_HW_ECDH_PARAM=y
+CONFIG_TLS_HW_ECDSA_VERIFICATION=y
```

Comment:
@jylee9613 Could you confirm this is right or not?

Reply:
These configs are related to mbedtls library, and they were enabled in commercial rtl8721csm. I think it's better to enable these configs.

Reply:
@jylee9613 these should be enabled in all defconfigs right?

Reply:
@jylee9613 , requesting confirmation on this please.

Reply:
Sorry for late apply. It should be enabled for onboarding.

Reply:
Done